1.fee_bumping/fee_bumping.md

```diff
@@ -321,11 +321,11 @@ their CPFP system:
   do the same process when creating a 3rd or 4th generation transaction to pay
   for its ancestors.
 - the Bitcoin Core mining algorithm will only consider packages of up to 25
-  transactions or 101vkB. HBE therefore needs to make sure they're not creating
+  transactions or 101vkB. HBE, therefore, needs to make sure they're not creating
```

Comment:
Again, I don't think these are needed. I'm with the Chicago Manual of Style 5.69 on this one: https://english.stackexchange.com/a/40241/83566